addons/account/static/tests/section_and_note_tests.js

```diff
@@ -89,6 +89,8 @@ QUnit.module('section_and_note', {
         // editing section should be input
         $tr1 = form.$('tr.o_data_row:eq(1)');
         await testUtils.dom.click($tr1.find('td.o_data_cell'));
+        await testUtils.nextTick(); // wait for discard
+        $tr1 = form.$('tr.o_data_row:eq(1)');
```

Comment:
I still don't understand. There's no row to discard here, we just pass the first row in edition and then the second one

Reply:
clicking on the second row unselects the first with discardIfNotDirty option set to true